app/controllers/application_controller.rb

```diff
@@ -1,8 +1,17 @@
 class ApplicationController < ActionController::Base
+  include Pundit
```

Comment:
Do you have an `ApiController` ? because you don't want to respond with a JSON if you are having Authorization Problems with a controller that renders HTML